### PR TITLE
Added blog and video to README

### PR DESCRIPTION
**Pull request by py-ranoid**
**Related issues**
None

**Describe the proposed solution**
Added blog and video links to README for newcomers to understand TransmogrifAI better

**Describe alternatives you've considered**
None

**Additional context**
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Use TransmogrifAI if you need a machine learning library to:
 * Build machine learning models without getting a Ph.D. in machine learning
 * Build modular, reusable, strongly typed machine learning workflows
 
+To understand TransmogrifAI better, check out these 
+- [Implementing AutoML Techniques at Salesforce Scale](https://databricks.com/session/implementing-automl-techniques-at-salesforce-scale), a talk by [@tovbinm](https://github.com/tovbinm)
+- [Automated Machine Learning for Structured Data](https://engineering.salesforce.com/open-sourcing-transmogrifai-4e5d0e098da2), a blog by [@snabar](https://github.com/snabar)
+
+Skip to [Quick Start and Documentation](#quick-start-and-documentation).
```
Comment (tovbinm) [inline README.md]: This line is a duplicate.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Use TransmogrifAI if you need a machine learning library to:
 * Build machine learning models without getting a Ph.D. in machine learning
 * Build modular, reusable, strongly typed machine learning workflows
 
+To understand TransmogrifAI better, check out these 
+- [Implementing AutoML Techniques at Salesforce Scale](https://databricks.com/session/implementing-automl-techniques-at-salesforce-scale), a talk by [@tovbinm](https://github.com/tovbinm)
+- [Automated Machine Learning for Structured Data](https://engineering.salesforce.com/open-sourcing-transmogrifai-4e5d0e098da2), a blog by [@snabar](https://github.com/snabar)
```
Comment (tovbinm) [inline README.md]: `a blog post by`

File: README.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ Use TransmogrifAI if you need a machine learning library to:
 * Build machine learning models without getting a Ph.D. in machine learning
 * Build modular, reusable, strongly typed machine learning workflows
 
+To understand TransmogrifAI better, check out these
```
Comment (tovbinm) [inline README.md]: `To understand the motivation behind TransmogrifAI check out these:`

**Comment by tovbinm** [comment]
@mayukhb @snabar @leahmcguire should we be adding these links here or not, WDYT?

**Comment by py-ranoid** [comment]
Made the changes you suggested.

**Pull request merged**